content/backmatter.tex

```diff
@@ -181,6 +187,16 @@ \chapter{Undefined Behavior in OpenSHMEM}\label{sec:undefined}
 \end{itemize}
 \tabularnewline
 \hline
+Concurrently using active-set-based and team-based collectives & Concurrently
+using the default team on an active-set-based collective with the default
+team in a team-based collective, for instance inside a parallel region,
+results in undefined behavior.\tabularnewline
```

Comment:
I think this case actually is trying to cover collectives that implicitly operate on the world team (e.g., `shmem_barrier_all`, `shmem_malloc`) with explicit team-based operations on `SHMEM_TEAM_WORLD`. The phrasing "using the default team on an active-set-based collective" doesn't make sense to me. (We do distinctly note the three cases of team-based, active-set-based, and implicit-team-world in the _Collectives_ introduction.)

```suggestion
Concurrent use of a team & Teams are not thread-safe objects.
Concurrent use of a team from multiple threads results in undefined
behavior.  Such a situation can arise when one thread is calling a
team-implicit collective (e.g., \FUNC{shmem\_barrier\_all}), which
implicitly operate on the world team, and another calls a team-based
collective (e.g., \FUNC{shmem\_broadcastmem}). \tabularnewline
```

Reply:
Yes, exactly, I meant to describe usage that's implicitly on the world team.  Thanks for the clarification - this is much better.

Another possible typo "operate" -> "operates".  I'll push that fix.